### ISSUE TITLE
If only one of camera or microphone are blocked, both are shown as blocked

### ISSUE THREAD

**Issue by swemail** [daily-co/react-native-daily-js#37]
Hello and thanks again for a great service and library!

# Expected behavior

Device allowing either only microphone or only video can use the one allowed

# Describe the bug (unexpected behavior)

When device denies access to video only both
```javascript
daily.participants().local?.tracks.audio.blocked?.byPermissions // true
daily.participants().local?.tracks.video.blocked?.byPermissions // true
```
are `true`

Same is true for denying microphone only.

Same on both iOS and Android

**Comment by gewfy** [comment]
@filipi87 This is probably a react native specific issue right? Or should we report this to `daily-js` too? Sort of a blocker for us.

**Comment by filipi87** [comment]
Hi @swemail , 

Thanks for reporting this.

I have been able to reproduce It here, and we have already merged a fix for It. It will be available on the next release of `react-native-daily-js`.

We are also working to improve, if you are building an app that just needs audio or video, to just request the permission that is needed, but this change will come in another future release.

**Comment by filipi87** [comment]
Hi @gewfy , 

About your question, It was not exactly just a `react-native` only issue. But for `daily-js` we already have a way to make It work. We have a flag, `enableIndependentDevicePermissionPrompts`.

Note that for `enableIndependentDevicePermissionPrompts` to work as you expect, you should have mic and cam forced off when you start using the call object.

```
const newCallObject = Daily.createCallObject({
  startVideoOff: true,
  startAudioOff: true,
  dailyConfig: {
    enableIndependentDevicePermissionPrompts: true
  }
});
```

This way the browser will ask one permission each time, and you will be able to accept or deny each one.

We **don't** need to use this flag for **react-native**.

**Comment by gewfy** [comment]
Makes sense @filipi87! We want the owner to be able to see which participants have denied permissions and to what. That's why we needed this functionality 😃 

Thanks for fixing this 🙌

**Comment by swemail** [comment]
> Hi @swemail ,
> 
> Thanks for reporting this.
> 
> I have been able to reproduce It here, and we have already merged a fix for It. It will be available on the next release of `react-native-daily-js`.
> 
> We are also working to improve, if you are building an app that just needs audio or video, to just request the permission that is needed, but this change will come in another future release.

Awesome news @filipi87! Thanks for fixing this!

**Comment by swemail** [comment]
I have confirmed that it works now, great work @filipi87! This issue could be closed now. Not sure if you like reporters to do that or you do it yourself?

**Comment by filipi87** [comment]
Fixed already available in the version `0.36.0` of [react-native-daily-js](https://www.npmjs.com/package/@daily-co/react-native-daily-js)

Thanks @swemail and @gewfy for reporting this.
